### PR TITLE
Transfer OP tokens to SmartEscrow

### PR DESCRIPTION
**Pull request by anikaraghu**
Updating the task to just send all the tokens to the SmartEscrow since Agora no longer provides partial delegation, and tokens will begin disbursing in 1 month, so we will just wait until then to participate in governance.

### CODE REVIEW

**Comment by cb-heimdall** [comment]
<h1>🟡 Heimdall Review Status</h1>
<table>
<tr>
<th>Requirement</th>
<th>Status</th>
<th>More Info</th>
</tr>
<tr>
<td>Reviews</td>
<td>
🟡
<code>0/1</code>
</td>
<td>
<details>
<summary>Denominator calculation</summary>
<table>
<tr>
<td>
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><code>1</code> if user is bot</td>
<td>0</td>
</tr>
<tr>
<td><code>1</code> if user is external</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>.codeflow.yml</code>
</td>
<td>1</td>
</tr>
<tr>
<td>
Additional review requirements
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><strong>Max</strong></td>
<td><strong>0</strong></td>
</tr>
</table>
</details>
</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>CODEOWNERS</code>
</td>
<td>0</td>
</tr>
<tr>
<td>Global minimum</td>
<td>0</td>
</tr>
<tr>
<td><strong>Max</strong></td>
<td>
<strong>
1
</strong>
</td>
</tr>
</table>
</details>
</td>
<td>1</td>
</tr>
<tr>
<td><code>1</code> if commit is unverified</td>
<td>0</td>
</tr>
<tr>
<td><strong>Sum</strong></td>
<td><strong>1</strong></td>
</tr>
</table>
</details>
</td>
</tr>
</table>

File: mainnet/2024-07-30-transfer-op/.env [hunk 1]
```diff
@@ -0,0 +1,11 @@
+OPTIMISM_RPC_URL=https://mainnet.optimism.io
+
+OP_COMMIT=3580bf1b41d80fcb2b895d5610836bfad27fc989
+BASE_CONTRACTS_COMMIT=a147139671c09923f78ae46a6ebedc91209bb076
+
+OP_TOKEN=0x4200000000000000000000000000000000000042
+NESTED_SAFE=0x0a7361e734cf3f0394B0FC4a45C74E7a4Ec70940
```
Comment (anikaraghu) [inline mainnet/2024-07-30-transfer-op/.env:7]: https://optimistic.etherscan.io/address/0x0a7361e734cf3f0394B0FC4a45C74E7a4Ec70940

File: mainnet/2024-07-30-transfer-op/.env [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+OP_MULTISIG=0x2501c477D0A35545a387Aa4A3EEe4292A9a8B3F0
```
Comment (anikaraghu) [inline mainnet/2024-07-30-transfer-op/.env:8]: https://optimistic.etherscan.io/address/0x2501c477D0A35545a387Aa4A3EEe4292A9a8B3F0

File: mainnet/2024-07-30-transfer-op/.env [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+CB_MULTISIG=0x6e1DFd5C1E22A4677663A81D24C6BA03561ef0f6
```
Comment (anikaraghu) [inline mainnet/2024-07-30-transfer-op/.env:9]: https://optimistic.etherscan.io/address/0x6e1DFd5C1E22A4677663A81D24C6BA03561ef0f6

File: mainnet/2024-07-30-transfer-op/.env [hunk 4]
```diff
@@ -0,0 +1,11 @@
+OPTIMISM_RPC_URL=https://mainnet.optimism.io
+
+OP_COMMIT=3580bf1b41d80fcb2b895d5610836bfad27fc989
+BASE_CONTRACTS_COMMIT=a147139671c09923f78ae46a6ebedc91209bb076
+
+OP_TOKEN=0x4200000000000000000000000000000000000042
```
Comment (anikaraghu) [inline mainnet/2024-07-30-transfer-op/.env:6]: https://optimistic.etherscan.io/address/0x4200000000000000000000000000000000000042

**Review by 0x00101010 — APPROVED** [review]
Looks good, confirmed all the env vars for contract addresses and numbers are correct.

**Pull request merged**